beater/config/auth.go

```diff
@@ -113,6 +113,8 @@ func defaultAnonymousAgentAuth() AnonymousAgentAuth {
 		Enabled:    false,
 		AllowAgent: []string{"rum-js", "js-base"},
 		RateLimit: RateLimit{
+			// TODO: Cloud is using the deprecated rum.rate_limit, should we update it there?
+			// https://github.com/elastic/cloud-assets/blob/master/stackpack/apm/config/apm-server.yml#L94-L99
```

Comment:
Yes, we should use `apm-server.auth.anonymous.rate_limit` for 8.0+.

Reply:
oops, that slipped through the cracks when I worked on updating the deprecated settings in cloud-assets (https://github.com/elastic/cloud-assets/pull/834). Do you want me to take care of it?

Reply:
That'd be good, if you have time

Reply:
https://github.com/elastic/cloud-assets/pull/860

Reply:
I commented on that PR, but I think we also need to align the rate limit defaults we have in code and the integration. What's in the integration looks wrong. I think I made an error when trying to align with the cloud template, we should probably update the integration to use the defaults we have in code at the moment.

Reply:
https://github.com/elastic/apm-server/pull/6715